.repo/sample-list-generator/README.md

```diff
@@ -0,0 +1,147 @@
+# Sample List Generator
+
+## Overview
+
+It's a script that generates `./extension-samples.json` with the list of all the samples available. Currently, this JSON will be provided to [developer.chrome.com](https://developer.chrome.com) for generating a list page containing all the samples. This allows developers to quickly find the sample they want to reference.
+
+## How to use
+
+### Install dependencies
+
+```bash
+npm install
+```
+
+### Run prefetch script (optional)
+
+The prefetch script will generate a list of all the available extension apis on [developer.chrome.com](https://developer.chrome.com/docs/extensions/reference) and save it to `./extension-apis.json`.
+
+The file `./extension-apis.json` will be committed so you don't need to run this script unless you want to update the list.
+
+```bash
+npm run prepare-chrome-types
+```
+
+### Run the generator
+
+```bash
+npm start
+```
```

Comment:
Explain how to run tests

Reply:
Done. https://github.com/GoogleChrome/chrome-extensions-samples/pull/963/commits/b316e888c2e88c6fab2c51b1a102138b4c49d6ee